vsts/pipelines/templates/_releaseJobTemplate.yml

```diff
@@ -29,34 +29,6 @@ jobs:
       
   - template: _releaseStepTemplate.yml
 
-- job: Release_PlatformSDKs
-  displayName: Publish SDKs from dev to prod storage account
```

Comment:
thanks for updating it, will the pipeline `Oryx-PublishSdkToProd` be automatically triggered each time after a CI build or it's scheduled every Monday

Reply:
yes @qianz2  thats the plan